sdks/python/setup.py

```diff
@@ -148,8 +148,8 @@ def get_version():
     'google-cloud-datastore>=1.7.1,<1.8.0',
     'google-cloud-pubsub>=0.39.0,<0.40.0',
     # GCP packages required by tests
-    'google-cloud-bigquery>=1.6.0,<1.7.0',
-    'google-cloud-core>=0.28.1,<0.30.0',
+    'google-cloud-bigquery>=1.6.0,<1.18.0',
+    'google-cloud-core>=0.28.1,<2',
```

Comment:
Did you mean `<1`?

Reply:
No I meant 2.

It does not look like 0.x to 1.x change was breaking. Beam could support both 0.x and 1.x ranges. 2 is not released. Added <2 to protect against future changes.